### PR TITLE
Fix and rewrite all the styles of the app to have a better order and to reuse them

### PR DESCRIPTION
**Pull request by mcass19**

### CODE REVIEW

File: app/assets/stylesheets/retrospective.scss [hunk 1]
```diff
@@ -1,83 +1,36 @@
...
     font-size: 20px;
-    font-weight: bold; //acceptance criteria
   }
 
-  .timer-last-minutes {
-    color: #FF0000;
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 20px;
-    font-weight: bold; //aceptance criteria
+  .to-discuss-div {
```
Comment (FLarra) [inline app/assets/stylesheets/retrospective.scss]: It's not common to add the type element in the css class. I'd try to think on an alternative to it!

**Comment by mcass19** [comment]
Working on this branch !!!

File: app/views/retrospectives/new.js.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 $('#button_timer').html("<%=j render('timer') %>");
+$('#to_discuss_section').removeClass('discuss_section hide').addClass('discuss_section');
```
Comment (fchavat) [inline app/views/retrospectives/new.js.erb]: Same as above with `discuss_section`, maybe it's to ensure that `discuss_section` is not added twice? jQuery's `addClass` already checks it 😃

File: app/assets/javascripts/timer.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ function executeTimer() {
     }
     timerToShow.text(minutesToPrint + ':' + secondsToPrint);
     if (minutes === 50) {
-      timerToShow.removeClass("timer").addClass("timer-last-minutes");
+      timerToShow.removeClass("timer").addClass("timer last-minutes");
```
[the remark this answers is not in the document]
Comment (mcass19) [reply app/assets/javascripts/timer.js]: There isn't, I've changed that in 4a6933c.

File: app/views/retrospectives/new.js.erb (hunk 2 above)
Comment (mcass19) [reply app/views/retrospectives/new.js.erb]: Great ! It is clearer in the way you propose, I've changed that in 4a6933c.

**Pull request merged**